test/base-compiler-tests.ts

```diff
@@ -247,24 +253,30 @@ describe('Compiler execution', function () {
     it('buildenv compiler without target/march', () => {
         const buildenv = new BuildEnvSetupBase(noExecuteSupportCompilerInfo, ce);
         buildenv.getCompilerArch().should.equal(false);
-        buildenv.compilerSupportsX86.should.equal(true);
+        (buildenv as any).compilerSupportsX86.should.equal(true);
```

Comment:
Assuming this is a workaround for compilerSupportsX86 being protected?

Reply:
I wouldn't be against making that particular one public

Reply:
I'll make it public